onnxruntime/core/session/onnxruntime_c_api.cc

```diff
@@ -1379,6 +1401,8 @@ static constexpr OrtApi ort_api_1 = {
     &OrtApis::SessionGetInputName,
     &OrtApis::SessionGetOutputName,
     &OrtApis::SessionGetOverridableInitializerName,
+    &OrtApis::SessionEndProfiling,
```

Comment:
And these, see https://github.com/microsoft/onnxruntime/pull/2927 for more info on where to put them.

Reply:
Fixed.